src/styles/style.css

```diff
@@ -0,0 +1,106 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import "./stars.css";
+@import "./reset.css";
+
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
```

Comment:
Create a separate file for 'card' BEM block
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/65a87087-e038-49d9-a293-353bb4d911b5)